quinn-proto/src/connection/mod.rs

```diff
@@ -3527,6 +3527,27 @@ impl Connection {
     pub fn current_mtu(&self) -> u16 {
         self.path.current_mtu()
     }
+
+    /// Upper bound for number of bytes in the next 1-RTT packet on the current path consumed by the
+    /// QUIC header and AEAD tag
+    ///
+    /// All other bytes in a packet are frames.
+    fn overhead_bound_1rtt(&self) -> usize {
```

Comment:
Don't love this name -- maybe `max_1rtt_overhead()`? Would also stop the initial docstrings sentence after "packet" and leave the details of "current path" and "consumed by" to a second sentence.